app/views/works/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<h1>Media</h1>
+<h3>Albums</h3>
+<table>
+<tr><td>Votes</td><td>Title</td><td>Created By</td><td>Published</td><td>Upvote</td></tr>
+<% @album.each do |work| %>
+<tr>
+  <td><%=work.votes.length%></td>
+  <td><%= link_to "#{work.title}", work_path(work.id) %></td><td>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?